App/src/pages/dossier/DossierBrowser.tsx

```diff
@@ -41,9 +43,13 @@ export default function DossierBrowser() {
         GetAllGroups()
             .then((res: MultiGroupResponseDTO) => {
                 setMyGroups(res.data);
+                if (res.data.length == 0) {
```

Comment:
I think this is a probably not what you meant, since this will show the toast when no groups are found rather than when no dossiers are found. Otherwise, looks good, so when you fix that feel free to merge.

Reply:
Good catch, just fixed it